openslides/motions/serializers.py

```diff
@@ -101,12 +101,43 @@ class WorkflowSerializer(ModelSerializer):
     Serializer for motion.models.Workflow objects.
     """
     states = StateSerializer(many=True, read_only=True)
-    first_state = PrimaryKeyRelatedField(read_only=True)
+    # The first_state is checked in the update() method
+    first_state = PrimaryKeyRelatedField(queryset=State.objects.all(), required=False)
```

Comment:
Do we really want the user to change the first state? I propose that the user can not change the first state but only modify it (rename etc.).

Reply:
I don't know. I think this should be possible. And currently it works ;)